src/scripts/get-all-files.ts

```diff
@@ -1,6 +1,8 @@
 import { promise } from "readdirp";
 
-const getAllFiles = (rootPath: string) => {
+export type ReaddirReturn = ReturnType<typeof promise>;
```

Comment:
This may seem a bit funky, it is.
For some reason readdirp keeps all their TS interfaces internal.
This seems like an antipattern and should be resolved upstream.

Reply:
Looks like this can cause an issue with generating typings files as part of the build
https://github.com/gitsome/json-schema-validators-typscript-interfaces/issues/14